.github/workflows/install_deps.sh

```diff
@@ -26,7 +26,6 @@ fi
 if [[ "$GH_OS" == "ubuntu-latest" ]]; then
   sudo apt-get -y install $GH_DEPS_UBUNTU
 elif [[ "$GH_OS" == "macos-latest" ]]; then
-  brew update
   # macos-latest already has both gcc and clang
```

Comment:
Might want to drop a comment here like

    # As of 2020-09-30, brew update sometimes times out on Github CI, so we don't do it here

and a link to a bug report/SO question, if there is any. (It sounded like you were trying to throw that ball into my court, but so far I've ignored it. ;) I don't really know how to file issues about Github CI; seems like maybe one would just post on https://github.community/search?q=%22brew%20update%22 ?)

Reply:
> It sounded like you were trying to throw that ball into my court, but so far I've ignored it.

Yes and no :). Much of the things I write here are also for myself, because memory/attention are unreliable.
I think https://github.community is just "one user helps another" type of deal. There was a special form for contacting Github support though. I'll see about filing a report later.